server/network/aoprotocol.py

```diff
@@ -697,9 +715,12 @@ def net_cmd_ct(self, args):
                 'CT',
                 '[' + self.client.area.abbreviation + ']' + self.client.name,
                 args[1])
-            database.log_room('ooc', self.client,
-                              self.client.area, message=args[1])
-
+            if(bool(self.server.config['log_chat'])):
+                database.log_room('ooc', self.client, self.client.area, message=args[1])
+            else:
+                time =  str(datetime.datetime.now().time())[:-7]
+                database.log_buffer(f'[{time}]'+'[OOC]' + f'[{self.client.area.abbreviation}] {self.client.char_name}' +
```

Comment:
Copy-pasted code. Consider having the timestamp be created inside the body of your `log_buffer` command.